tests/jvm/src/test/resources/example/Classes.scala

```diff
@@ -23,11 +23,44 @@ class C8/*<=classes.C8#*/(private[this] val x/*<=classes.C8#x.*/: Int/*=>scala.I
 class C9/*<=classes.C9#*/(private[this] var x/*<=classes.C9#x().*/: Int/*=>scala.Int#*/)
 
 object N/*<=classes.N.*/ {
-  val anonClass/*<=classes.N.anonClass.*/ = new C7/*=>classes.C7#*/(42) {
+  val anonClass/*<=classes.N.anonClass.*/ = new /*<=local8*/C7/*=>classes.C7#*/(42) {
```

Comment:
Weirdly enough this looks like a correct change, but not sure how it is influenced by this PR